calibr8/core.py

```diff
@@ -19,7 +19,7 @@
 from . import utils
 from .utils import pm
 
-__version__ = "7.0.0"
+__version__ = "7.1.0"
```

Comment:
Isn't that technically a breaking change?

Reply:
Yes, but not for the model functions. Realistically, no one is using calibr8 with PyMC v4 and I'd like to NOT follow this up with changes to all cmodel JSONs we have internally 🙄